oaknational/Oak-Web-Application
# ENG-489 feat: handle preview errors

## Description

- I'd wanted this work to result in the user (whoever is editing content on CMS) to be shown nicely displayed errors telling them what's wrong
- However, this wasn't possible, so this simply stops such errors being reported to BugSnag
- @mcky it turns out that the errors we were getting in BugSnag were valid errors (i.e. schema in OWA did't match config in Sanity). So maybe it's best to not merge this and instead to actually treat BugSnag errors from PreviewMode seriously?

### How to test

- locally you can change a schema and check a preview version and see that it doesn't attempt to report
